test/issues/0468.js

```diff
@@ -6,7 +6,15 @@ var yaml = require('../..');
 
 test('should not indent arrays an extra level when disabled', function () {
   /* eslint-disable max-len */
-  var output = yaml.dump([ { a:'a_val', b:'b_val' }, { a:'a2_val', expanded:true, items: [ { a:'a_a_val', b:'a_b_val' } ] } ], { noArrayIndent: true });
+  var output = yaml.dump(
+    [
+      { a:'a_val', b:'b_val' },
+      { a:'a2_val', expanded:true, items:
+        [
+          { a:'a_a_val', b:'a_b_val' }
+        ]
+      }
+    ], { noArrayIndent: true });
```

Comment:
```js
var output = yaml.dump(
  [
    {
      a: 'a_val',
      b:'b_val'
    },
    {
      a: 'a2_val',
      expanded:true,
      items: [
        {
          a: 'a_a_val',
          b: 'a_b_val'
        }
      ]
    }
  ],
  { noArrayIndent: true }
);
```

1. Change formatting to above please.
2. Is this sample really minimal required for this test? For example, i don't understand what field `expanded` does here.

Reply:
fixed